### PR TITLE
TRUNK-5403 Fixed and tested 1.5 and 1.6 DTDs

### PR DESCRIPTION
**Pull request by peytontanzillo**
## Description of what I changed

* I fixed the 1.5 DTD, as it had a few small syntax errors that prevented it from running in a command line and in testing.
* I created the 1.6 DTD with the conditionalResources element.
* I added unit tests for both the 1.5 and the 1.6 DTD files. These are located in api/src/test/java/org/openmrs/module/dtd/

## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-5403

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

- [x] I ran `mvn clean package` right before creating this pull request and

- [x] All new and existing **tests passed**.

There are two tests that currently fail, but only because it depends on http://resources.openmrs.org/doctype/config-x.x.dtd, which does not work (but will work when my DTD versions are uploaded).

- [x] My pull request is **based on the latest changes** of the master branch.
`

### CODE REVIEW

**Comment by dkayiwa** [comment]
Can you include the ticket id in your commit message as advised at? https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by peytontanzillo** [comment]
@dkayiwa The ticket ID has been put in the commit message.
Please let me know if you have any additional issues!

**Comment by peytontanzillo** [comment]
Hello! We're running into an issue with Travis where it says that we do not have the expected error:
```Failed to execute goal com.mycila:license-maven-plugin:3.0:check (default) on project openmrs-api: Some files do not have the expected license header -> [Help 1]```
Do you have an idea what this problem is?

Thanks!

**Comment by dkayiwa** [comment]
You can ask for help on openmrs talk

**Comment by peytontanzillo** [comment]
Alright, now that Travis is passing, may you please look at this pull request? @dkayiwa

File: api/src/main/resources/org/openmrs/module/dtd/config-1.5.dtd [hunk 1]
```diff
@@ -1,5 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dkayiwa) [inline api/src/main/resources/org/openmrs/module/dtd/config-1.5.dtd]: Did you remove this intentionally?

Comment (peytontanzillo) [reply api/src/main/resources/org/openmrs/module/dtd/config-1.5.dtd]: I did, from my understanding it was unnecessary. Do you want me to add it back to the dtd files?

Comment (peytontanzillo) [reply api/src/main/resources/org/openmrs/module/dtd/config-1.5.dtd]: It has been brought to my attention that the DTD files do, in fact, need the XML version and encoding line. Therefore, I have added the line back and pushed my changes again.

Comment (apilkington2) [reply api/src/main/resources/org/openmrs/module/dtd/config-1.5.dtd]: @dkayiwa besides that does the code look good?

File: api/src/main/resources/org/openmrs/module/dtd/config-1.5.dtd [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dkayiwa) [inline api/src/main/resources/org/openmrs/module/dtd/config-1.5.dtd]: I thought the ticket was not about the 1.5 config. So i would not expect any changes in this file.

**Pull request closed without merging**